docs/user-guide/side-bar.md

```diff
@@ -101,3 +105,14 @@ In order to have information about one layer only the user can select the layer
 
 <img src="../img/side-bar/layer.gif" class="ms-docimage"/>
 
+### Floating Identify Tool 
+
+On [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) the user can activate the **Floating Identify tool** instead of the default [Identify tool](side-bar.md#identify-tool) through which a mobile panel opens to retrieve information about an object on the map.
```

Comment:
In [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) the user can set the Identify tool in floating mode (**Floating Identify tool**) instead of having the default one available through a click on the map. In that case an identify popup will appears on the map as soon as the user hover over a layer in the map.